shop_stripe/payment.py

```diff
@@ -22,8 +24,12 @@ def get_payment_request(self, cart, request):
         """
         From the given request, add a snippet to the page.
         """
+        stripe.api_key = settings.SHOP_STRIPE['APIKEY']
+        stripe.api_version = settings.SHOP_STRIPE_API_VERSION
```

Comment:
This is a new setting I am adding to SHOP_STRIPE. I was debating on whether it should go into SHOP_STRIPE or not, and played it safe by having it separate.

Since it seems we would want to push these changes upstream eventually, and because it looks like adding that setting into SHOP_STRIPE won't break anything (at least for now), I'll make the change and add it to that dictionary.